api/core/schema_loader.py

```
@@ -162,3 +164,74 @@ async def list_databases(user_id: str, general_prefix: Optional[str] = None) ->
+                message="Database connected and schema loaded successfully",
+            )
+
+        return DatabaseConnection(
+            database_id="",
+            success=False,
+            tables_loaded=0,
+            message=last_message or "Failed to load database schema",
+        )
+
+    except (RedisError, ConnectionError, OSError) as e:
+        logging.exception("Error loading database: %s", str(e))
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)